src/main/java/org/jenkinsci/plugins/quarantine/QuarantinableJUnitResultArchiver.java

```diff
@@ -43,8 +43,8 @@ public QuarantinableJUnitResultArchiver(String testResults) {
 	@Override
 	public void perform(Run build, FilePath workspace, Launcher launcher,
 			TaskListener listener) throws InterruptedException, IOException {
-				
-		listener.getLogger().println(hudson.tasks.junit.Messages.JUnitResultArchiver_Recording());
```

Comment:
These were annoying. I also had to removed them when I was testing with new junit plugin locally

Reply:
yep had to use strings as these were marked internal in the updated `hudson.tasks.junit` classes.